orchagent/vnetorch.cpp

```diff
@@ -335,11 +335,20 @@ void VNetBitmapObject::recycleBitmapId(const string& vnet)
     }
 }
 
-uint32_t VNetBitmapObject::getFreeTunnelRouteTableOffset()
+uint32_t VNetBitmapObject::getFreeTunnelRouteTableOffset(IpPrefix ipPfx)
 {
     SWSS_LOG_ENTER();
 
-    for (uint32_t i = 0; i < tunnelOffsets_.size(); i++)
+    uint32_t offsetStart = VNET_ROUTE_FULL_MASK_OFFSET_MAX + 1;
```

Comment:
tunnelOffsets[VNET_ROUTE_FULL_MASK_OFFSET_MAX] will not be included in any range.
It should be `uint32_t offsetStart = VNET_ROUTE_FULL_MASK_OFFSET_MAX;`